include/RAJA/policy/openmp_target/forall.hpp

```diff
@@ -31,12 +31,23 @@ namespace omp
 /// OpenMP target parallel for policy implementation
 ///
 
+//template <size_t ThreadsPerTeam, typename Iterable, typename Func>
+//RAJA_INLINE void forall_impl(const omp_target_parallel_for_exec<ThreadsPerTeam>& exec,
+//                             Iterable&& iter,
+//                             Func&& loop_body)
+//{
+//  RAJA::resources::Resource res{RAJA::resources::Omp()};
+//  forall_impl(res, exec, iter, loop_body);
+//}
 template <size_t ThreadsPerTeam, typename Iterable, typename Func>
-// RAJA_INLINE void forall(const omp_target_parallel_for_exec<Teams>&,
-RAJA_INLINE void forall_impl(const omp_target_parallel_for_exec<ThreadsPerTeam>&,
-                             Iterable&& iter,
-                             Func&& loop_body)
+RAJA_INLINE resources::EventProxy<resources::Omp> forall_impl(resources::Omp &omp_res,
+                                                              const omp_target_parallel_for_exec<ThreadsPerTeam>&,
+                                                              Iterable&& iter,
+                                                              Func&& loop_body)
 {
+  char *omp_ptr;
+  omp_res.get_ptr_dev(omp_ptr);
```

Comment:
Is this used?

Reply:
Nope, Omp target resource support is broken. Removed.